src/microbe_stage/MicrobeStage.cs

```diff
@@ -300,6 +310,8 @@ public void SetupStage()
             }
         }
 
+        GD.Print(WorldSettings);
```

Comment:
Doesn't this print a ton of info? I think maybe always logging this isn't the best to keep log clutter down a bit...

Reply:
I wouldn't say it logs a lot, only three lines.

![image](https://user-images.githubusercontent.com/7430433/174793742-9415288a-39ec-4a1c-9394-dbf751cc3ba8.png)

I definitely think this needs to be logged somewhere, as otherwise we wouldn't have a record of the player's world settings if we just receive their logs. But if you'd prefer it to be logged somewhere different than main output, I can change that.

Reply:
Well we don't log the players overall settings either... But fine, let's have the settings in the output for now, I can always remove the registry load prints to reduce the amount of logs a debugging session prints.